packages/docs/src/pages/color-modes.mdx

```diff
@@ -28,13 +30,118 @@ In the `theme.colors` object, add a nested `modes` object that will contain keys
 }
 ```
 
-The colors defined at the root of the `colors` object will be accessible as `default`. All colors
-found in `colors.modes` will be referenced by their key. The above example will have two modes: `default` and `dark`.
+The colors defined at the root of the `colors` object will be accessible as
+`default`. All colors found in `colors.modes` will be referenced by their key.
+The above example will have two modes: `default` and `dark`.
+
+## Colors Object
+
+The `colors` object contains Custom CSS Properties
+
+```js
+{
+  colors: {
+    text: 'var(--theme-ui-colors-text)',
+    background: 'var(--theme-ui-colors-background)',
+    primary: 'var(--theme-ui-colors-primary)',
+  }
+}
+```
+
+## rawColors Object
+
+If you need to pass original value somewhere where CSS Properties (e.g. WebGL
+Canvas) won't work, use `rawColors`
+
+```js
+{
+  rawColors: {
+    text: '#000',
+    background: '#fff',
+    primary: '#07c',
+  }
+}
+```
+
+## Default colors
+
+The default colors are tranformed upon `colorMode` change. This allows for the
+use the "current" colors. `theme.colors.primary` returns the primary key of the
+current mode.
+
+We also add those default colors inside `rawColors.modes` under the `__default`
+key (or `initialColorModeName` value, if it is defined).
+
+```js
+{
+  initialColorModeName: 'light',
+  rawColors: {
+    primary: '#07c',
+    modes: {
+      // __default: {}, initialColorModeName is defined
+      light: { primary: '#07c' },
+      dark: { primary: '#0cf' }
+    }
+  }
+}
+```
+
+### Use specific modes
+
+#### With the `sx` prop
+
+```jsx
+export default (props) => (
+  <div
+    sx={(theme) => ({
+      color: theme.rawColors.modes?.dark?.text
+      bg: theme.rawColors.modes?.dark?.bg
+    })}
+  />
+)
+
+```
+
+#### With Theme UI context
+
+Use the `useThemeUI` hook to access the context object directly in a component.
+
+```jsx
+import { useThemeUI } from 'theme-ui'
+
+export default (props) => {
+  const context = useColorMode(null)
+  const { setColorMode } = context
+  const { rawColors: colors } = context.theme
+
+  return Object.entries(colors?.modes).map(([mode, values]) => ({
+    <Button
+      sx={{ bg: values.background, color: values.text }}
+      onClick={() => setColorMode(mode)}
+    >
+      {mode}
+    </Button>
+  }))
+}
+
+// OUTPUT
+
+<Button>light</Button>
+<Button>dark</Button>
+<Button>deep</Button>
+// ...
+```
+
+<Note>
+
+For more info, [see the migration guide](/migrating/#breaking-changes)
```

Comment:
Let's update with a specific section, so that this line doesn't become out of date

Reply:
I removed the note completely, the info on the page should be sufficient.

And I think that updating the docs with the content of the migration guide should be a PR of its own.